site/src/pages/WorkspacePage/Workspace.tsx

```diff
@@ -55,7 +55,7 @@ export interface WorkspaceProps {
   workspaceErrors: WorkspaceErrors;
   buildInfo?: TypesGen.BuildInfoResponse;
   sshPrefix?: string;
-  template?: TypesGen.Template;
+  template: TypesGen.Template;
```

Comment:
Why did we change this type?

Reply:
I noticed it didn't need to be conditional and see those as a liability so I just drive-by fixed it while I was reading the code.